src/lib/KQStream.ts

```diff
@@ -135,22 +147,78 @@ export class KQStream extends ProtectedEventEmitter<Events> {
             this.sendMessage('im alive', null);
             break;
         case 'playernames':
-            // Not sure what the values of the message mean,
-            // so just pass an empty object for now.
-            this.protectedEmit('playernames', {});
+            const playernames = parsers.playernames(parsedMessage.data);
+            this.protectedEmit('playernames', playernames);
             break;
         case 'playerKill':
-            const [x, y, by, killed] = parsedMessage.data.split(',');
-            const playerKill: PlayerKill = {
-                pos: {
-                    x: Number(x),
-                    y: Number(y)
-                },
-                killed: Number(killed),
-                by: Number(by)
-            };
+            const playerKill = parsers.playerKill(parsedMessage.data);
             this.protectedEmit('playerKill', playerKill);
             break;
+        // New events from beta 2018-08-21
+        case 'blessMaiden':
```

Comment:
maybe a dumb question, but is there a way to reuse these strings?  I see the type defined as `GameEvents`, but I have no idea how to reverse that relationship, e.g., (imagined) `GameEvents.BlessMaiden` instead of `'blessMaiden'`

Reply:
It can definitely be done, just need to use an appropriate pattern. Happy to revisit it later since we have good coverage with unit tests.